baseplate/thrift/message_queue/message_queue.thrift

```diff
@@ -0,0 +1,27 @@
+namespace go reddit.message_queue
+namespace py message_queue.thrift
+namespace java com.reddit.baseplate.message_queue
+
+struct CreateResponse {}
+
+struct PutResponse {}
+
+struct GetResponse {
+    1: binary value;
+}
+
+exception ThriftTimedOutError {}
+
+service RemoteMessageQueueService {
+    CreateResponse create_queue(
```

Comment:
> only major thing is something I think I missed in the initial review and when we were talking about this -- why do we need to support the queue server having multiple queues? (aka with the create_queue thrift rpc call). Is there a use case that I'm missing?

the posix queues are identified by their name, so a single application _could_ create and manage multiple queues separately. I haven't seen this in practice in any applications (but I also haven't checked thoroughly). I tried to mimic the posix queue's behavior closely, but if we want to enforce that there is only one queue I am ok with changing that - it would simplify things a bit. I think that the queue name is also unnecessary if there is only one queue.

Reply:
ok - removed create_queue entirely. we can always add it back if we find a use case for multiple queues